exonum/src/storage/base_index.rs

```diff
@@ -261,6 +261,8 @@ fn assert_valid_name<S: AsRef<str>>(name: S) {
 mod tests {
     use super::*;
 
+    static INDEX_NAME: &str = "test_index_name";
```

Comment:
FYI: this string is seemingly unused. I wonder if there is a way to make the CI builds fail because of this.